docs/src/__examples__/Stepper/DEFAULT.tsx

```diff
@@ -25,12 +25,6 @@ export default {
         { name: "maxValue", type: "number", defaultValue: 10 },
         { name: "step", type: "number", defaultValue: 1 },
         { name: "disabled", type: "boolean", defaultValue: false },
-        {
-          name: "size",
-          type: "select",
-          options: ["small", "normal"],
-          defaultValue: "small",
-        },
```

Comment:
I've noticed that this is not really a problem of the docs, but of the component itself. The prop is documented on the README but it actually does not exist on the component